interfaces/builtin/mnt.go

```diff
@@ -0,0 +1,51 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2018 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+const mntSummary = `allows access to anything mounted in /mnt`
```

Comment:
As suggested by @jdstrand in the forum, I'm also leaning towards adding this to removable-media. This directory was the typical place for removable media before /media showed up, and in both cases the sort of content we can have mounted under them is undefined and inconsistent in terms of structure. So forcing users to think about the two independently just feels like a nuisance.

The name "mnt" is also way too vague. So if we choose to not merge them, we'll at least need to think about something that more clearly defines what this is. I can't think of anything good right now.

Reply:
I'll fix the other things and then think about any other better name. If I cannot come up with anything then yeah, let's add that to `removable-media`